sdks/python/apache_beam/transforms/environments.py

```diff
@@ -595,6 +595,7 @@ def _python_sdk_capabilities_iter():
   yield common_urns.protocols.LEGACY_PROGRESS_REPORTING.urn
   yield common_urns.protocols.WORKER_STATUS.urn
   yield 'beam:version:sdk_base:' + DockerEnvironment.default_docker_image()
+  yield common_urns.sdf_components.TRUNCATE_SIZED_RESTRICTION.urn
```

Comment:
Lets not add this URN until we are sure with internal testing with Dataflow just in case we find that we need to fix/change something.